company/okrs/2020_q1.md

```diff
@@ -88,28 +88,18 @@ Build out vision for product and run customer validation discussions
    1. Web KR: At least four customers are using campaigns in production
    1. Web KR: All campaign types have been used by customers in production
 
-1. **Core Services: Build features that accelerate customers through pipeline**
-   1. Core Services KR: Shorten the time from "intent to try Sourcegraph" to "whole org is using Sourcegraph" (onboarding experience)
-   1. Core Services KR: Ensure new features (Automation, ACLs, etc) are accessible and easily tryable by customers once they go GA
-
-1. **Core Services: Search has feature parity with OpenGrok**
-   1. Core Services KR: Implement AND query support
-   1. Core Services KR: Implement OR query support
-   1. Core Services KR: Implement NOT query support
-
-1. **Core Services: Existing customers are cared for**
-   1. Core Services KR: Each iteration work plan includes prioritised customer issues.
-   1. Core Services KR: Each team mate works on relevant customer issues every iteration.
+1. **Core Services: Search gains new capabilities which are adopted by customers**
+   1. Core Services KR: Support AND/OR/NOT operators on file contents.
+   1. Core Services KR: Write at least 2 blog posts about new search features.
+   1. Core Services KR: Structural search can be ran at scale (~30k repositories).
+   1. Core Services KR: At least four customers are running structural searches.
+   1. Core Services KR: At least four customers are running AND/OR/NOT queries.
 
 1. **Core Services: Automation is polished and adopted by customers**
-   1. Core Services KR: Write blog post about automation.
-   1. Core Services KR: At least four customers are running campaigns.
-   1. Core Services KR: Implement [stacking diffs](https://docs.google.com/document/d/1m6WH47vl-DFynfEdjpGez2pj7bTO3MRFAPpDgEDlvFI/edit).
-
-1. **Core Services: Structural search is polished and adopted by customers**
-   1. Core Services KR: Write blog post about structural search.
-   1. Core Services KR: At least four customers are running structural searches.
-   1. Core Services KR: Structural search can be ran at scale (~30k repositories).
+   1. Core Services KR: Document Automation and write a blog post about it.
+   1. Core Services KR: Automation is enabled by default and not behind a feature flag.
+   1. Core Services KR: Run 3 user tests with Sourcegraph colleagues, collect issues and fix them.
+   1. Core Services KR: 3 different customers have together run at least 10 campaigns.
```

Comment:
Approve! 👍 

Tagging @lguychard here because Web & Core Services should be in sync here.

Reply:
The KRs look good. Re: ownership, what should we do here? Duplicating automation OKRs between web and core services does not make sense to me. These KRs are not core services-specific and web would be working towards them as well through our work on automation. What do you think of @nicksnyder's proposal to have product own this OKR (https://github.com/sourcegraph/about/pull/422/files#r365303011)?

Reply:
> What do you think of @nicksnyder's proposal to have product own this OKR (https://github.com/sourcegraph/about/pull/422/files#r365303011)?

That makes a lot of sense to me. We'd still work towards it and help product achieve this, mostly by working on the things on the roadmap and by fixing the things from the user tests, etc.

Reply:
@nicksnyder @christinaforney @tsenart since web and core services should be in sync here, what is the final takeaway here? Should this be listed as a "split ownership" OKR (`**Core services, web**`), or should it be owned by product as suggested by @nicksnyder in https://github.com/sourcegraph/about/pull/422/files#r365303011 ?

Reply:
The indirect ownership here feels off to me. This is a broader question beyond just Automation planning and as we continue to grow our cross-functional teams will require more collaboration. This almost feels like there is a need to have an Automation specific OKR that is owned by individuals on core and web. I know this is a bit of a regression back to having an Automation team, but this is still a big enough focus that requires two teams of effort.

Thoughts on creating a single Automation objective that is owned by both the core and web teams? Both teams would review the OKR in team meetings.

Reply:
I also thought about just creating an automation OKR, and since you mentioned it, I think it is a pragmatic solution for now. It avoids duplication in the OKRs and there is a set of people who clearly own it.

> Isn't that the same as both team having the same OKR? It's just how we display it that changes, right?

Yes.

Reply:
I pushed two changes to address this:

https://github.com/sourcegraph/about/pull/422/commits/84805a2796011dedbca29d222f5db46484a778a3
https://github.com/sourcegraph/about/pull/422/commits/a725aaedfd2611a650d24ee2a69f9c5cda05e58c